README.md

```diff
@@ -37,6 +37,60 @@ render(helloTemplate('Kevin'), document.body);
   </a>
 </p>
 
+## Getting Started
+
+Lit is based on ES6 Modules. If you're using NPM in your project, installing and using Lit is straightforward:
+
+```bash
+> npm install lit-html
+```
+
+```html
+<!DOCTYPE html>
+<html>
+  <head>
+    <script
+        type="module"
+        src="node_modules/lit-html/lit-html.js"></script>
+  </head>
```

Comment:
Does this create globals you can use to access `html` and `render`? I've only used the `import {html, render} ...` form.

Reply:
Yah, where do exported methods go?